demisto_sdk/commands/lint/linter.py

```diff
@@ -1601,15 +1612,28 @@ def _get_docker_images_for_lint(
 
             self._check_native_image_flag(docker_image_flag)
 
-            if native_image := self._get_native_image_name_from_config_file(
-                docker_image_flag
-            ):
+            supported_section = docker_image_flag
```

Comment:
I'm not sure about the naming of this var 'supported_section' - what is the meaning here?

Reply:
so, this is mainly used for providing `_get_native_image_name_from_config_file` an argument. 
This function "Gets a native docker image flag and returns its mapped native image name from the 'docker_native_image_config.json' file."

when using native target, we want all the native:dev *supported* images to lint on, which is different than the "docker_image_flag" (in our case native:target)

So instead of calling it docker_image_flag, I want the supported section from config file
I agree that the naming is a little confusing, I'm changing to `image_support`, let me know if you think of a better name for this variable.